INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 3.0.0 must be defined `k8s.io/apimachinery` override in Gopkg.toml
+
+```
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "3.0.0"
+
+[[override]]
```

Comment:
Can you change it to a `constraint`?

Dep's [doc](https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md) says: a constraint "Must not conflict with the constraint entries declared in any other project's manifest".

The user's project probably depends on `k8s.io/apimachinery` directly. In the future, `client-go` will have it's own `Gopkg.toml`, and it's constraint of `k8s.io/apimachinery` might conflict with user's. In that case, `dep` will emit an error to indicate the conflict if user defines a `constraint` here, and will silently ignore the conflict if user defines a `override` here. The former is better.